src/index.js

```diff
@@ -1,7 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
 import * as serviceWorker from './serviceWorker';
```

Comment:
Consider adding new lines between groups of imports.

Reply:
/fxd